ui/src/templates/components/TemplateImportOverlay.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {PureComponent} from 'react'
+import {withRouter, WithRouterProps} from 'react-router'
+import {connect} from 'react-redux'
+
+// Components
+import ImportOverlay from 'src/shared/components/ImportOverlay'
+
+// Actions
+import {
+  createTemplate as createTemplateAction,
+  setTemplatesStatus as setTemplatesStatusAction,
+} from 'src/templates/actions'
+
+// Types
+import {AppState, Organization} from 'src/types/v2'
+import {RemoteDataState} from 'src/types'
+
+interface DispatchProps {
+  createTemplate: typeof createTemplateAction
+  setTemplatesStatus: typeof setTemplatesStatusAction
+}
+
+interface StateProps {
+  org: Organization
+}
+
+interface OwnProps extends WithRouterProps {
+  params: {orgID: string}
+}
+
+type Props = DispatchProps & OwnProps & StateProps
+
+class TemplateImportOverlay extends PureComponent<Props> {
+  public render() {
+    return (
+      <ImportOverlay
+        onDismissOverlay={this.onDismiss}
+        resourceName="Template"
+        onSubmit={this.handleImportTemplate}
+      />
+    )
+  }
+
+  private onDismiss = () => {
+    const {router} = this.props
+
+    router.goBack()
+  }
+
+  private handleImportTemplate = () => async (
+    importString: string
+  ): Promise<void> => {
+    const {createTemplate} = this.props
+    const {setTemplatesStatus} = this.props
+    try {
+      const template = JSON.parse(importString)
+      await createTemplate(template)
+      setTemplatesStatus(RemoteDataState.NotStarted)
```

Comment:
is there a reason that the status is set to NotStarted rather than Done here?

Reply:
This is how importOverlay clears cache.